admin.widget.class.php

```diff
@@ -133,10 +142,16 @@ public function wpsp_admin_page()
         <?php
     }
 
+    public function admin_page_header($arg) {
+      $image_url = $this->asset_url('assets/logo-40.png');
+      printf('<div class="sp-heading"><img src="%s" alt="SparkPost"> &nbsp;&nbsp; <h2>%s</h2></div>', $image_url, 'SparkPost');
+    }
+
     public function admin_page_init()
     {
+        wp_register_style('sp-admin-css', $this->asset_url( 'assets/styles.css'));
```

Comment:
the examples i followed did this in two step. it may be better to register them all at once and then include (enqueue) where necessary. however, in our case we've one file so may not have that benefit.